gcsfs/tests/test_manyopens.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""
+Test helper to open the same file many times.
+"""
+from __future__ import print_function
+import sys
+import gcsfs
+
+def run():
```

Comment:
This does not look like a pytest test - you intention is that it should only be run on real GCS connections? It would not work with VCR recording anyway, which would presumably not return any errors (unless there happened to be errors when they were recorded).

Reply:
This is not a pytest. It's for manual testing. I don't know where to put it, open to suggestions.